### DISCUSSION TITLE
Sale Cost

### DISCUSSION THREAD

**Discussion by blakewintermute** [inventree/InvenTree#2871] [Q&A] [answered]
I feel as though I've looked at everything and I can't figure this out.

Where would I define a Sales Cost? Sales Cost according to the Wiki being the expected price to receive from a sale of a part to a customer.

How would I access that Sale Cost from the rest api?

Where do I define Price Information so that the price window doesn't state "No pricing information is available for this part." ?

I know these must seem like really basic questions, but I am at a loss. 

Thanks for the assistance.

**Comment by matmair** [comment]
Hi there @blackravensail !
I will need a bit more information before I can answer in detail. Do you want to set the price in general (that would be in the part then) or for a specific stock item once it is sold (that would be in the specific line of the item in a sales order).
Regards Matthias

**Reply by blakewintermute** [reply]
Hi @matmair ! Thanks again for your help.

I am trying to set the _Sale Cost_ (trying to use the terms as they're defined in the documentation) as it is associated with an individual part, so setting the "price in general".

I am also inquiring about how to access that _Sale Cost_ via the RESTAPI for any given part. Looking at the /api/part endpoint, it doesn't seem to be included in the json block describing each part, and I can't get the /api/price/sale-price/ endpoint to return anything besides an empty list.

Best,

Blake

**Comment by SchrodingersGat** [comment, marked as answer]
@blackravensail looks like there is a bug in current master which is hiding the "Pricing" tab on the "Part" window. Fix incoming here: https://github.com/inventree/InvenTree/pull/2875

Once that PR is merged, you should be able to access pricing information via the "Pricing" tab as shown below:

![image](https://user-images.githubusercontent.com/10080325/165268494-b8befa2b-de86-4239-932a-203247578269.png)
